packages/slider/test/slider-handle-upgrade.test.ts

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { expect, fixture, html } from '@open-wc/testing';
+import '../sp-slider.js';
+import { Slider } from '../src/Slider.js';
+
+describe('Slider', () => {
```

Comment:
Nit: can you use a longer name to differentiate this from the tests in https://github.com/adobe/spectrum-web-components/blob/main/packages/slider/test/slider.test.ts#L28 Maybe just `Slider - upgrade order` or something.

Reply:
Good idea 👍